administrator/components/com_admin/sql/updates/mysql/5.2.2-2024-11-11.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE `#__finder_links` MODIFY `route` VARCHAR(500);
```

Comment:
```suggestion
ALTER TABLE `#__finder_links` MODIFY `route` VARCHAR(500) NOT NULL;
```
It seems MySQL resets the NOT NULL constraint when doing a MODIFY.